python/ray/data/tests/test_tfrecords.py

```diff
@@ -455,6 +455,27 @@ def test_read_tfrecords(
     assert np.array_equal(df["string_empty"][0], np.array([], dtype=np.bytes_))
 
 
+@pytest.mark.parametrize("num_cpus", [1, 2, 4])
+def test_read_tfrecords_ray_remote_args(
+    ray_start_cluster,
+    tmp_path,
+    num_cpus,
+):
+    import tensorflow as tf
+
+    cluster = ray_start_cluster
+    cluster.add_node(num_cpus=num_cpus)
+    example = tf_records_empty()[0]
+    path = os.path.join(tmp_path, "data.tfrecords")
+    with tf.io.TFRecordWriter(path=path) as writer:
+        writer.write(example.SerializeToString())
+    ray_remote_args = {"num_cpus": num_cpus}
+    ds = read_tfrecords_with_tfx_read_override(
+        paths=[path],
+        ray_remote_args=ray_remote_args,
+    ).take_all()
```

Comment:
I think mainly you just need a mock, something similar to https://github.com/anyscale/rayturbo/blob/b7e33915b6bfd0362af36ef988756f0953386245/python/ray/data/tests/test_delta_sharing.py#L268-L280

Reply:
it seems a private repo which I cannot access. could you provide a accessible link for the example? thanks!

Reply:
oh shoot sorry, updated

Reply:
Thanks! just updated the test with mock obj as well. please take a look when you have time.